packages/api-headless-cms/src/crud/system.crud.ts

```diff
@@ -34,8 +35,14 @@ interface CreateSystemCrudParams {
 export const createSystemCrud = (params: CreateSystemCrudParams): CmsSystemContext => {
     const { getTenant, getLocale, storageOperations, context, getIdentity } = params;
 
-    const onBeforeInstall = createTopic<BeforeInstallTopicParams>();
-    const onAfterInstall = createTopic<AfterInstallTopicParams>();
+    const onSystemBeforeInstall = createTopic<BeforeInstallTopicParams>(
+        "cms.onSystemBeforeInstall"
```

Comment:
`onBeforeSystemInstall` if we want to stick strictly to the pattern would be better